build/scripts/build.ps1

```diff
@@ -300,6 +300,38 @@ function Build-ExtraSignArtifacts() {
         Write-Host "Building CompilersCodeAnalysis Swix"
         Run-MSBuild "DevDivVsix\CompilersPackage\Microsoft.CodeAnalysis.Compilers.swixproj"
 
+        $vsLanguages = "chs", "cht", "csy", "fra", "deu", "ita", "jpn", "kor", "plk", "ptb", "rus", "esn", "trk"
+
+        Write-Host "Building CompilersCodeAnalysis Resource Swixes"
+        foreach ($language in $vsLanguages) {
+            Run-MSBuild "DevDivVsix\CompilersPackage\Microsoft.CodeAnalysis.Compilers.Resources.swixproj" "/p:Lang=$($language)"
+        }
+
+        Write-Host "Building VisualStudioSetup Resources Swixes"
+        foreach ($language in $vsLanguages) {
+            Run-MSBuild "DevDivVsix\RoslynVisualStudioSetup\Microsoft.CodeAnalysis.VisualStudio.Setup.Resources.swixproj" "/p:Lang=$($language)"
+        }
+
+        Write-Host "Building InteractiveComponents Resources Swixes"
+        foreach ($language in $vsLanguages) {
+            Run-MSBuild "DevDivVsix\RoslynVisualStudioInteractiveComponents\Microsoft.CodeAnalysis.VisualStudio.InteractiveComponents.Resources.swixproj" "/p:Lang=$($language)"
+        }
+
+        Write-Host "Building ExpressionEvaluatorPackage Resources Swixes"
+        foreach ($language in $vsLanguages) {
+            Run-MSBuild "DevDivVsix\MicrosoftCodeAnalysisExpressionEvaluator\Microsoft.CodeAnalysis.ExpressionEvaluator.Resources.swixproj" "/p:Lang=$($language)"
+        }
+
+        Write-Host "Building Roslyn.Compilers.Extension Resources Swixes"
+        foreach ($language in $vsLanguages) {
+            Run-MSBuild "DevDivVsix\RoslynCompilersExtension\Roslyn.Compilers.Extension.Resources.swixproj" "/p:Lang=$($language)"
+        }
+
+        Write-Host "Building Roslyn.Compilers.Extension Resources Swixes"
+        foreach ($language in $vsLanguages) {
+            Run-MSBuild "DevDivVsix\RoslynVisualStudioDiagnosticsWindow\Roslyn.VisualStudio.Diagnostics.Window.Resources.swixproj" "/p:Lang=$($language)"
+        }
```

Comment:
How much time does this add to the builds? Seems fairly significant. Is it possible for us to reduce the number of MSBuild invocations here? Possibly by creating an uber project of some sort.